packages/react-jss/src/createHoc.js

```diff
@@ -274,18 +269,21 @@ export default function createHOC<
     }
   }
 
-  if (isThemingEnabled || injectMap.theme) {
-    const ThemeConsumer = (theming && theming.context.Consumer) || ThemeContext.Consumer
-
-    // eslint-disable-next-line no-inner-declarations
-    function ContextSubscribers(props) {
-      return <ThemeConsumer>{theme => <Jss theme={theme} {...props} />}</ThemeConsumer>
-    }
-
-    ContextSubscribers.InnerComponent = InnerComponent
-
-    return ContextSubscribers
+  return function ContextSubscriber(props) {
```

Comment:
This is basically a component user will see in the devtools, right?

Reply:
Yes.

Reply:
Can we avoid having this additional component in each HOC?

Reply:
Nope, not possible.

Reply:
hmm, what about having the Consumer inside of Jss Component?

Reply:
You will still have to render Consumer, but you won't have ContextSubscriber component

Reply:
Won't work because we need the context in the lifecycle methods

Reply:
Ok, lets rename to JssContextSubscriber

Reply:
or JssContextConsumer